guides/scripts/generate-hammer-reference.sh

```diff
@@ -154,7 +155,7 @@ cat "$SRC" | while read line ; do
             id=[id=\"$id_core\"]
             heading=`echo -e $line | sed 's/^### hammer/==/'`
```

Comment:
According to our mod doc guidelines, all modules should begin with level 1 headers. 

Would it be possible to change this to level 1 and add 'echo -e "include::modules/$filename[leveloffset=+2]\n" >>$asmb_file'?

Reply:
Potentially yes, but the problem is that there are 5 levels of commands. If we structure the modules for 5 levels, the nesting of the modules will be too deep (>3 nesting levels of sections). We don't want that either.

Perhaps we could think a bit deeper about the design of the reference before we make any changes to the script.